### PR TITLE
[SYCL][CI] Avoid format check crash on change first lines in a file

### PR DESCRIPTION
**Pull request by vladimirlaz**
Fix issues reported in intel/llvm-test-suite#709 to avoid a crash on modifying first/end line in the file.

The check passed in the testing https://github.com/intel/llvm-test-suite/runs/4758725035?check_suite_focus=true

### CODE REVIEW

File: SYCL/DeviceLib/cmath_test.cpp [hunk 1]
```diff
@@ -1,5 +1,3 @@
-// FIXME unsupported on windows (opencl) until fix of libdevice fails
-// UNSUPPORTED: windows && opencl
```
Comment (alexbatashev) [inline SYCL/DeviceLib/cmath_test.cpp]: This seems unrelated

**Review by alexbatashev — DISMISSED** [review]
LGTM otherwise

**Review by alexbatashev — APPROVED** [review]

File: SYCL/DeviceLib/cmath_test.cpp (hunk 1 above)
Comment (vladimirlaz) [reply SYCL/DeviceLib/cmath_test.cpp]: it was added for testing new action (this change exposed crash).
